dags/destinations/gads_oci.py

```diff
@@ -186,7 +188,7 @@ def _get_valid_and_invalid_conversions(
 
       click_conversion.conversion_value = float(conversion.get("conversion_value", ""))
       click_conversion.conversion_date_time = conversion.get("conversion_date_time", "")
-      click_conversion.currency_code = conversion.get("currency_code", _DEFAULT_CURRENCY_CODE)
+      click_conversion.currency_code = conversion.get("currency_code", False) or _DEFAULT_CURRENCY_CODE
```

Comment:
This is a tricky one. 

If the currency_code is defined as "", the default value would not be used but this would generate an "invalid currency code error". So that new version is kind of a "coalesce" for the dict get method.

Does that make sense? I will add a comment to document this.